spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::NetworkManager do
+  context "ems" do
+    it "does not support network creation" do
+      ems = FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud)
+      expect(ems.supports_ems_network_new?).to eq(false)
+    end
+  end
+
+  context "singleton methods" do
+    it "returns the expected value for the description method" do
+      expect(described_class.description).to eq('IBM Power Systems Virtual Servers')
```

Comment:
Almost, still needs ' Network' at the end: `IBM Power Systems Virtual Servers Network`
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager.rb#L47